scripts/apps/authoring/authoring/directives/SendItem.tsx

```diff
@@ -7,7 +7,7 @@ import {AuthoringWorkspaceService} from '../services/AuthoringWorkspaceService';
 import {appConfig, extensions} from 'appConfig';
 import {IExtensionActivationResult, IArticle} from 'superdesk-api';
 import {ITEM_STATE} from 'apps/archive/constants';
-import {confirmQuickPublish} from '../services/quick-publish-modal';
+import {confirmQuickPublish as confirmPublish} from '../services/quick-publish-modal';
```

Comment:
Can you rename the original function instead of creating an alias?

Reply:
> Can you rename the original function instead of creating an alias?

Thanks, i will do changes as per your suggestion.